sources/engine/Xenko.VirtualReality/DeviceEnums.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+namespace Xenko.VirtualReality
+{
+    public enum DeviceState
```

Comment:
Why moving the enum into a different file?

Reply:
before, it was one enum in a file with the same name as the enum (DeviceState). since i added a second enum concerning the Device i thought, instead of adding a second file, i'd put them both in the same file and renaming the file to a more generic name, ie. DeviceEnums.cs.

please let me know if you prefer one file per enum and i'll change it accordingly.

Reply:
Don't mind so much with so small enums, but so far most of the code is one file per enum, so might be better to keep consistency.